packages/admin/src/pages/build-application/[applicationId]/[sectionId]/[questionId]/edit/question-content/index.page.tsx

```diff
@@ -100,6 +101,8 @@ const QuestionContent = ({
             defaultChecked={optionalRadioDefault()}
           />
 
+          <input type="hidden" name="version" value={version} />
```

Comment:
Doesnt need to be an input - can grab is in the `handleRequest` function from the `pageData` prop